bindings/pyroot/pythonizations/python/ROOT/_hist.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
```

Comment:
Please don't import numpy globally here. You need to import it locally in the functions, otherwise you force always loading these math libraries that numpy loads (see failing tests).

Python is very good at caching imported modules, so moving the import inside the function bodies won't impact the performance (just mentioning in case this was the reason you were making this import global).